inlang/source-code/sdk/package.json

```diff
@@ -50,6 +50,7 @@
 		"debug": "^4.3.4",
 		"dedent": "1.5.1",
 		"deepmerge-ts": "^5.1.0",
+		"messageformat": "4.0.0-7",
```

Comment:
should we pin to this version or use a ^ ?

Reply:
Let's leave it pinned for now. We don't really care about future updates, we aren't trying to replicate MF2 1-to-1. We just want to express our AST more concisely. 
Unless we find a serious bug in the package I don't see us ever needing to update.

Reply:
better to remove dependency entirely.